content/guides/weird_characters.adoc

```diff
@@ -0,0 +1,846 @@
+= The Weird and Wonderful Characters of Clojure
+James Hughes
+2017-05-27
+:type: guides
+:toc: macro
+
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+
+[]
+====
+This is a reference collection of characters used in Clojure that are difficult to "google".
+Descriptions are sourced from various blogs, https://stackoverflow.com[StackOverflow],
+http://en.wikibooks.org/wiki/Learning_Clojure[Learning Clojure], and the
+http://clojure.org/documentation[official Clojure docs] -- sources attributed
+where necessary. Sections are not in any particular order, but related items
+are grouped for ease. This guide is a copy of http://twitter.com/kouphax[James Hughes]
+brilliant https://yobriefca.se/blog/2014/05/19/the-weird-and-wonderful-characters-of-clojure/[blog post] of the same name.
+====
+[[dispatch]]
+== `#` - Dispatch macro
+
+You'll see this macro character beside another e.g. `\#(` or `#"`.
+// " Comment needed for emacs to behave.
+This topic will act as a bit of a preamble before looking at your specific case.
+
+`#` is a reader macro that tells the Clojure reader (the thing that takes a
+file of Clojure code and parses it for
+consumption by the compiler) to go and look at another *read table*
+for the definition of the next character - in essence this allows
+extending default reader behaviour.
+
+Clojure doesn't provide support for creating reader macros, but it is possible
+through http://briancarper.net/blog/449/[a bit of hackery].
+
+If you se `#` *at the end* of a symbol, then it is used to automatically
+generate a new symbol. This is useful inside macros to keep macro specifics
```

Comment:
This explanation and example are not very good and seem like they should be either near the other macro characters like ~ or at the end of the guide where we talk about other suffix characters, rather than here (and this should just refer).